src/js/profile/chart.js

```diff
@@ -91,7 +91,7 @@ export class Chart extends Observable {
     getValuesFromSubindicators = () => {
         let arr = [];
         for (const [label, subindicator] of Object.entries(this.subindicators)) {
-            let count = subindicator.count;
+            let count = isNaN(subindicator.count) ? 0 : subindicator.count;
```

Comment:
this is not the right place to make this change.
subindicator is a class `SubIndicator` and they have a `MISSING_VALUE` constant. 
For now let's change that from `N/A` to `0` and we should make a pin and refactor this, so we can change the MISSING_VALUE for a subindicator.

Reply:
done